Example4_advanced.py

```diff
@@ -68,7 +68,7 @@ def main():
         'max_features': [1, 5, 10, 20, 50, 100, 1000, 10000],
         'max_leaf_nodes': [None, 30, 400]}
 
-    hcai.advanced_random_forest_classifier(
+    hcai.random_forest_classifier(
         trees=500,
         scoring_metric='accuracy',
         hyperparameter_grid=knn_hyperparameters,
```

Comment:
knn usually stands for k nearest neighbor (a different algorithm from random forest). Does it stand for something else here?